app/views/about.html

```diff
@@ -22,15 +22,9 @@ <h1>About</h1>
 
       <h2>Support</h2>
 
-      <p>The NHS prototype kit is maintained by NHS England. If you've got a question or need support you can:</p>
+      <p>View our <a href="/support">support</a> section.</p>
```

Comment:
Could drop both lines to save the awkwardness of cross-linking?

Reply:
Yep agreed - going to see if this makes the commit stick.